app/Http/Controllers/Frontend/Admin/TripController.php

```diff
@@ -2,24 +2,14 @@
 
 namespace App\Http\Controllers\Frontend\Admin;
 
-use App\Enum\HafasTravelType;
-use App\Http\Controllers\Backend\Transport\ManualTripCreator;
-use App\Http\Controllers\Backend\Transport\ManualTripCreator as TripBackend;
-use App\Models\HafasOperator;
 use App\Models\Trip;
-use App\Models\Station;
-use App\Models\Stopover;
-use Carbon\Carbon;
-use Illuminate\Http\RedirectResponse;
-use Illuminate\Http\Request;
-use Illuminate\Validation\Rules\Enum;
 use Illuminate\View\View;
 
 class TripController
 {
 
     public function renderTrip(string $id): View {
-        $trip = Trip::with(['checkins'])
+        $trip = Trip::with(['checkins', 'polyline', 'polyline.parent'])
```

Comment:
`polyline` is already loaded with `polyline.parent`.

```suggestion
        $trip = Trip::with(['checkins', 'polyline.parent'])
```